<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>237</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Uninformed-Informed search  algorithms</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>End (x,y)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="dest_x">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Start (x,y)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="source_x">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="source_y">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="dest_y">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="set_source">
        <property name="text">
         <string>Set (x,y)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="set_dest">
        <property name="text">
         <string>Set (x,y)</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextEdit" name="algorithm_output"/>
    </item>
    <item>
     <widget class="QPushButton" name="btn_process">
      <property name="text">
       <string>Process</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>360</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFiles">
    <property name="title">
     <string>Uninformed</string>
    </property>
    <addaction name="actionBreadth_first_search"/>
    <addaction name="actionDepth_first_searc"/>
    <addaction name="actionIterative_deepening_search"/>
   </widget>
   <widget class="QMenu" name="menuInformed">
    <property name="title">
     <string>Informed</string>
    </property>
    <addaction name="actionBest_first_search"/>
    <addaction name="actionA_search"/>
    <addaction name="actionDijkstra"/>
   </widget>
   <addaction name="menuFiles"/>
   <addaction name="menuInformed"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionBreadth_first_search">
   <property name="text">
    <string>Breadth first search</string>
   </property>
  </action>
  <action name="actionDepth_first_searc">
   <property name="text">
    <string>Depth first search</string>
   </property>
  </action>
  <action name="actionIterative_deepening_search">
   <property name="text">
    <string>Iterative Deepening Search</string>
   </property>
  </action>
  <action name="actionBest_first_search">
   <property name="text">
    <string>Best first search</string>
   </property>
  </action>
  <action name="actionA_search">
   <property name="text">
    <string>A * search</string>
   </property>
  </action>
  <action name="actionDijkstra">
   <property name="text">
    <string>Dijkstra</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
